remote/remote.go

```diff
@@ -865,6 +865,12 @@ func (i *Image) ManifestSize() (int64, error) {
 	return i.image.Size()
 }
 
+func (i *Image) AnnotateRefName(refName string) error { return nil }
```

Comment:
We could throw a not implemented error, since in theory you might want to add this annotation for a remote image. It would be nice to know that it doesn't work